app/src/main/java/com/sopt/now/data/Search/GitHubService.kt

```diff
@@ -0,0 +1,10 @@
+package com.sopt.now.data.Search
+
+import com.sopt.now.presentation.main.search.repo.Repo
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface GitHubService {
+    @GET("users/2hyunjinn/repos")
+    fun listRepos(): Call<List<Repo>>
```

Comment:
```suggestion
    @GET("users/{userId}/repos")
    fun listRepos(@Path("userId") userId : String): Call<List<Repo>>
```
지금의 방식대로 하면 항상 API 호출 시 2hyunjinn에 대한 Repository만 가져올 것 같다고 예상되는데요, 위의 방식으로 하면 유동적으로 호출 API URL 을 바꿀 수 있을 것 같습니다.